docs/admin/install.rst

```diff
@@ -0,0 +1,265 @@
+
+Copy Journalist Interface details 
+---------------------------------
+
+SecureDrop Workstation connects to your SecureDrop instance's API via the Journalist Interface. In order to do so, it will need the Journalist Interface address and authorization info. This can be copied from the Admin Workstation via the following steps:
+
+- Connect the Admin Workstation USB to a USB port on the Qubes computer, then use the devices widget in the upper right panel to attach it to the ``vault`` VM. There will be 3 listings for the USB in the widget: one for the base USB, one for the Tails partition on the USB, labeled ``Tails``, and a 3rd unlabeled listing, for the persistent volume. Choose the third listing.
+
+- In the the ``vault`` file manager, select **+ Other Locations**., then click the persistent volume's listing in the right panel. It will be named ```N GB encrypted``, where N is the size of the persistent volume. Enter the Admin Workstation persistent volume passphrase to unlock and mount it. 
+
+- Copy the Journalist Interface configuration file to `dom0`. If your SecureDrop instance uses v3 onion services, use the following command:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault "cat /run/media/user/TailsData/Persistent/securedrop/install_files/ansible-base/app-journalist.auth_private" > /tmp/journalist.txt
+
+  If your instance uses legacy v2 onion services, use the following command instead:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault "cat /run/media/user/TailsData/Persistent/securedrop/install_files/ansible-base/app-journalist-aths" > /tmp/journalist.txt
+
+- Verify that the `/tmp/journalist.txt` file on ``dom0`` contains valid configuration information using the command ``cat /tmp/journalist.txt`` in the administration terminal.
+
+- In the ``vault`` file manager, select **+ Other Locations** and eject the TailsData volume, then disconnect the Admin Workstation USB. Shut down the ``vault`` VM using the Qube widget in the upper right panel.
+
+
+Download and install SecureDrop Workstation
+-------------------------------------------
+
+With the key and configuration available in ``dom0``, you're ready to set up SecureDrop Workstation:
+
+- First, re-enable the network connection using the network manager widget.
+
+- Next, start a terminal in the network-attached ``work`` VM, via **Q > Domain:work > work: Terminal**.
+
+- In the ``work`` terminal, run the following commands to download and add the SecureDrop signing key, which is needed to verify the SecureDrop Workstation package:
+
+  .. code-block:: sh
+
+    gpg --keyserver hkps://keys.openpgp.org --recv-key "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
+    gpg --armor --export 22245C81E3BAEB4138B36061310F561200F4AD77 | sudo tee /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation
+
+- In the ``work`` terminal, create a file ``/etc/yum.repos.d/securedrop-temp.repo`` with the following contents:
```

Comment:
We should specify that `sudo` is required (implicit if we give example with `vi`, or `nano` after installing)

Reply:
:heavy_check_mark: